notebooks/text_classification.ipynb

```diff
@@ -842,8 +841,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "train_data['text_length'] = train_data['text'].apply(lambda x: len(x)) # get the text length\n",
-    "train_data['text_num_tokens'] = train_data['text'].apply(lambda x: len(x.split()))  # split the text by spaces and get the length of the resulting list to calculate the number of tokens"
+    "train_data['text_length'] = train_data['text'].str.len()  # get the text length (https://pandas.pydata.org/docs/reference/api/pandas.Series.str.len.html)\n",
+    "train_data['text_num_tokens'] = train_data['text'].str.split()  # split the text by spaces and get the length of the resulting list to calculate the number of tokens (https://pandas.pydata.org/docs/reference/api/pandas.Series.str.split.html)"
```

Comment:
Here is the only place where I moved from apply to native methods (with respective links)